stable/spring-cloud-data-flow/templates/database-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if not .Values.mysql.enabled }}
```

Comment:
`mysql` references here would be generified if going the route of "external database" vs. "external mysql", including the secret key